### PR TITLE
Add Info plist, version, podspec and CHANGELOG changes for the 1.6.0 release.

### PR DESCRIPTION
**Pull request by tirodkar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,32 @@
...
+
+```
+Baseline: [fdda61c]
+   + [fdda61c]: Fix issue with Rakefile
+```
+
+### Compatibility
+* Requires iOS 8 as the minimum deployment target.
+* The EarlGrey gem runs out of the box for Swift 3.0 and Swift 2.3.
+* Has been tested for support till iOS 10.01 on devices and simulator.
```
Comment (sid-github) [inline CHANGELOG.md]: 10.0.1?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,32 @@
...
+
+### Bug Fixes
+* Fixed CocoaPods issue with using EarlGrey as a module in Swift projects.
+* Minor documentation and syntax fixes.
+
+### Enhancements
+* Moved failure handler from a global variable to a thread local storage, like NSAssertionHandlers.
+* Exposed the angle for a GREYPinchAction.*
+* Added EarlGreyExample CocoaPods project to travis, deintegrating it for a clean setup.*
```
Comment (sid-github) [inline CHANGELOG.md]: setup.

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,9 +46,12 @@ Baseline: [f3ee931]
 ```
 
 ### Compatibility
-* Handle bit-dependent CGFloat with bit-dependent trig function
+* Requires iOS 8 as the minimum deployment target.
+* The EarlGrey gem runs out of the box for Swift 3.0 and Swift 2.3.
+* Has been tested for support till iOS 10.01 on devices and simulator.
```
Comment (sid-github) [inline CHANGELOG.md]: 10.0.1?

**Comment by khandpur** [comment]
Holding this off as we have a few more fixes to land in the next release.

**Comment by sid-github** [comment]
Closing this as I will be sending another one out.

**Pull request closed without merging**